proto/ibc/applications/interchain_accounts/host/v1/tx.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package ibc.applications.interchain_accounts.host.v1;
+
+option go_package = "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/types";
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/interchain_accounts/host/v1/host.proto";
+
+// Msg defines the 27-interchain-accounts/host Msg service.
+service Msg {
```

Comment:
and the equivalent one for `services` https://github.com/cosmos/ibc-go/pull/3630